pipelines/jobs/configurations/jdk20_pipeline_config.groovy

```diff
@@ -0,0 +1,185 @@
+class Config20 {
+  final Map<String, Map<String, ?>> buildConfigurations = [
+        x64Mac    : [
+                os                  : 'mac',
+                arch                : 'x64',
+                additionalNodeLabels: 'macos10.14',
+                additionalTestLabels: [
+                        openj9      : '!sw.os.osx.10_11'
```

Comment:
It would be nice to rip out the OpenJ9 specific sections from the configurations here if possible but lets merge for now. Maybe we can do this in a follow-up PR?

Reply:
I believe these configurations are still in use, so I would not want to see them removed as it would break some things.